documentation/library_guide/overview.rst

```diff
@@ -37,8 +37,12 @@ and use the ``std`` namespace.
 Prerequisites
 =============
 
-C++11 is the minimal version of the C++ standard that |onedpl_short| requires. That means, any use of |onedpl_short|
-requires at least a C++11 compiler. Some APIs of the library may require a higher version of C++.
+Since oneDPL 2021.6 C++17 is the minimal version of the C++ standard that |onedpl_short| requires. 
```

Comment:
Since |onedpl_short| 2021.6, C++17 is the minimal supported version of the C++ standard. That means, any use of |onedpl_short| may require a C++17 compiler. While some APIs of the library may accidentally work with earlier versions of the C++ standard, it is no more guaranteed.

Reply:
Thanks, applied changes